WheelOfFortune/Game.cs

```diff
@@ -50,43 +50,29 @@ public class Game
         /// </summary>
         public void StartGame()
         {
-            Console.WriteLine("Press any button to start the game!");
-            Console.ReadKey(true);
-            AddPlayer();
+            Prompt.WelcomeMessage();
 
-            Random random = new();
-            int index = random.Next(0, allPuzzles.Count);
-            string nextPuzzle = allPuzzles[index];
-            Puzzle puzzle = new(nextPuzzle);
-            CurrentPuzzle = puzzle;
+            string person = Prompt.CreatePlayer();
+            Player player = new Player(person);
+            Players.Enqueue(player);
 
-            StartTurn();
-        } 
-
-        /// <summary>
-        /// A method that initializes Player and its properties
-        /// </summary>
-        public void AddPlayer()
-        {
-            Console.WriteLine("Hey there! Welcome to Wheel of Fortune! Before we begin, what is your first name?");
-            string name = Console.ReadLine();
-
-            // ** user validation check
-            // originally had it as an if conditon but realized while loop would be better in the event Player repeatedly does not pass in input for name
-            while (string.IsNullOrEmpty(name))
+            char addNewPlayerResponse = Prompt.AddAdditionalPlayer();
+            while (addNewPlayerResponse == 'y')
             {
-                Console.WriteLine("Vanna White would like to know your name. Please input your name once more so we can play the game!");
-                name = Console.ReadLine();
+                person = Prompt.CreatePlayer();
+                player = new Player(person);
+                Players.Enqueue(player);
+                addNewPlayerResponse = Prompt.AddAdditionalPlayer();
```

Comment:
A stylistic comment (and I'm not sure if one is better than the other) - alternatively, the while loop condition could also be written as `while (Prompt.AddAdditionalPlayer() == 'y')`, allowing you to get rid of lines 59 and 65. But I also think that having variable names can sometimes add to the readability of the code, so up to you on what you think is better!

Reply:
Also stylistic - you could combine the first call for the player and additional players by using a `do...while` loop I believe

Reply:
If you decide to update the variable names above - you might also want to change these accordingly!

Reply:
I like both ideas. Give me a moment to make those changes